docs/migrate.md

```diff
@@ -4,6 +4,10 @@ This document records manual migrations that are necessary to apply when upgradi
 Sourcegraph versions. All manual migrations between the version you are upgrading from and the
 version you are upgrading to should be applied (unless otherwise noted).
 
+## 3.8
+
+If you're deploying Sourcegraph into a non-default namespace, refer to [base/prometheus/README.md#namespace](../base/prometheus/README.md#namespace) for further configuration instructions.
```

Comment:
Doesn't grafana also need the same instructions?

Reply:
done